samples/sqs/native-integration/Sqs_6/Receiver/Program.cs

```diff
@@ -7,13 +7,15 @@ class Program
     static async Task Main()
     {
         Console.Title = "Samples.Sqs.SimpleReceiver";
-        var endpointConfiguration = new EndpointConfiguration("Samples.Sqs.SimpleReceiver");
+        var endpointConfiguration = new EndpointConfiguration("Samples.Sqs.SimpleReceiver");        
         endpointConfiguration.EnableInstallers();
-        endpointConfiguration.UseTransport(new SqsTransport());
+
+        var transport = new SqsTransport();
+        transport.DoNotWrapOutgoingMessages = true;
```

Comment:
would make it consistent with other snippets

```suggestion
        var transport = new SqsTransport
        {
            DoNotWrapOutgoingMessages = true
        };
```